frontend/javascripts/oxalis/model/volumetracing/volumelayer.js

```diff
@@ -212,9 +211,8 @@ class VolumeLayer {
 
   getCircleVoxelIterator(position: Vector3, boundings?: ?BoundingBoxType): VoxelIterator {
     const state = Store.getState();
-    const zoomFactor = getZoomValue(state.flycam);
     const { brushSize } = state.userConfiguration;
-    const scaledBrushSize = brushSize * zoomFactor;
+    const scaledBrushSize = brushSize;
```

Comment:
`scaledBrushSize` is a bit misleading now, I think this line can simply be removed